distribution/changelog.txt

```diff
@@ -1,18 +1,24 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: OpenGL renderengine added.
 - Feature: Ability to rotate map elements with the tile inspector.
-- Feature: Add cheat to disable littering.
-- Feature: Add ride console command for diagnostics and changing vehicle type.
+- Feature: Added cheat to disable littering.
+- Feature: Addded ride console command for diagnostics and changing vehicle type.
```

Comment:
There is precedent for using simple present tense "add" for features, I'd leave this as is.